db/migrate/20190729145805_add_whitehall_import.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class AddWhitehallImport < ActiveRecord::Migration[5.2]
+  def change
+    create_table :whitehall_imports do |t|
+      t.bigint :whitehall_document_id
+      t.json :payload
+      t.uuid :content_id
+      t.string :import_status
```

Comment:
I'd suggest naming this `state` to be consistent with existing fields we have such as: https://github.com/alphagov/content-publisher/blob/master/db/schema.rb#L273 and https://github.com/alphagov/content-publisher/blob/master/db/schema.rb#L105

It'd also be good for this to have enums defined for what the values could be in the model: https://github.com/alphagov/content-publisher/blob/master/app/models/file_attachment/asset.rb#L16-L19